src/components/Popover/Popover.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import {
+  Close,
+  Content,
+  PopoverContentProps,
+  PopoverProps,
+  Portal,
+  Root,
+  Trigger,
+} from "@radix-ui/react-popover";
+import { twMerge } from "tailwind-merge";
+import { IconButton } from "../IconButton";
+
+const Popover = Root;
+
+const PopoverTrigger = Trigger;
+
+interface PopoverHeaderProps {
+  className?: string;
+  title: string;
+}
+const PopoverHeader = ({ className, title }: PopoverHeaderProps) => {
+  return (
+    <div
+      className={twMerge("flex justify-between items-center pb-4", className)}
```

Comment:
Honestly I don't like this `pb-4`. Why only on the bottom? I think we should remove it and let the developer define how the spacing will be. The use could add `space-y-4` to `PopoverContent` if he wants to have some bottom spacing, for example. 

I know that we are defining this because we are adding `p-4` on `PopoverContent` and I'm a little against it too 😄 

But that's just an opinion. What do you think @ElRodrigote @Diogomartf ?

Reply:
But can't we wrap this inside a component and export it has `PopoverHeader`?

```
<div className="flex justify-between items-center mx-4 pb-4">
    <p className="text-lg text-text-high-em font-bold">Settings</p>
    <Close asChild>
        <IconButton name="cross" size="md" variant="ghost" />
    </Close>
</div>
```

In this case we just need a `title` prop and a `className` prop to override styles.

Reply:
Yes we can! In this case we will need to use this Header inside the `<PopoverContent>`, I thought we wanted a different thing, I'll create few samples now and push them.